esp/scm/ws_config2.ecm

```diff
@@ -199,6 +199,7 @@ ESPstruct AttributeType
     string     CurrentValue("");
     string     DefaultValue("");
     string     CodeDefault("");
+    bool       IsPresentInEnvironment;
```

Comment:
Should this have a default value?

Reply:
It is always assigned a value in the code.